### ISSUE TITLE
execute tests in less than 50 minutes

### ISSUE THREAD

**Issue by javigomez** [joomla-projects/GSOC-Webdriver_system_tests_for_CMS#58]
This is going to be a challenge:

![screen shot 2014-07-28 at 11 21 09](https://cloud.githubusercontent.com/assets/1375475/3718609/5ba46cd2-1638-11e4-89dc-f056150ea589.png)

But we have this:
http://blog.travis-ci.com/2012-11-28-speeding-up-your-tests-by-parallelizing-them/

And this:
http://codeception.com/docs/12-ParallelExecution

**Comment by puneet0191** [comment]
The solution is CodeeeeeCeptionnnnnnn :p

**Comment by puneet0191** [comment]
Yes this still remains a Challenge, working on a getting a new server to help us in running System Tests on every PR
